examples/rust-parser/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
```

Comment:
I guess the Rust parser can be removed, since the other projects more than cover the needed information.